tests/unit/s2n_early_data_io_api_test.c

```diff
@@ -130,6 +129,7 @@ int main(int argc, char **argv)
 
     /* Test s2n_send_early_data */
     {
+        /* clang-format bug 48305 https://bugs.llvm.org/show_bug.cgi?id=48305 work around */;
```

Comment:
It's ridiculous to put this on so many tests. I'd prefer we just didn't format our unit tests until this bug is resolved.

Reply:
It's been unresolved since clang version 11. The issue with just not formatting the tests is that it effects the rest of the outer code block. Which for some of the test files means we might as well exclude them all together.

Reply:
I kinda agree its ridiculous; but I'm really not sure if there is a better option. I'd prefer for the code to be formatted than not (or severely incorrectly).

One alternative option is to remove the surrounding code blocks we have a lot of code that follows the following basic outline:
```
   {
     {...}
     {...}
     {...}
   }
```
Semantically the outer braces are meaningless because there is nothing declared in them, but not their children. 

On the other hand they are very useful for code navigation and I'd strongly prefer not to get rid of them.

Reply:
I'll exclude the files for now. But in the long run I think we need to find a way to deal with this.